indigo_app/views/tasks.py

```diff
@@ -90,6 +90,8 @@ def get_context_data(self, **kwargs):
         context['frbr_uri'] = self.request.GET.get('frbr_uri')
         context['task_groups'] = Task.task_columns(self.form.cleaned_data['state'], context['tasks'])
 
+        context["taxonomy_toc"] = json.dumps(TaxonomyTopic.get_toc_tree(self.request.GET))
```

Comment:
in places.py you don't use json.dumps -- are the uses different?

Reply:
Removed `json.dumps`